modules/06 Angular/Assignments/12/answer12/src/app/actor/actor.model.ts

```diff
@@ -0,0 +1,6 @@
+export interface Actor {
```

Comment:
Use class instead, this is a kind of best practice to use class instead of interface in case you interact not with server entity